pkg/api/api.go

```diff
@@ -2,14 +2,14 @@ package api
 
 import (
 	"github.com/redhat-developer/app-services-cli/pkg/api/ams/amsclient"
-	strimziadminclient "github.com/redhat-developer/app-services-cli/pkg/api/strimzi-admin/client"
+	kafkainstanceclient "github.com/redhat-developer/app-services-sdk-go/kafkainstance/apiv1internal/client"
 	kafkamgmtclient "github.com/redhat-developer/app-services-sdk-go/kafkamgmt/apiv1/client"
 )
 
 // API is a type which defines a number of API creator functions
 type API struct {
 	Kafka          func() kafkamgmtclient.DefaultApi
 	ServiceAccount func() kafkamgmtclient.SecurityApi
-	TopicAdmin     func(kafkaID string) (strimziadminclient.DefaultApi, *kafkamgmtclient.KafkaRequest, error)
+	KafkaAdmin     func(kafkaID string) (kafkainstanceclient.DefaultApi, *kafkamgmtclient.KafkaRequest, error)
```

Comment:
Thank you for this. FYI - in my branch I'm using registryInstance. We call this KafkaAdmin so it is ok, unless we want to group them somewhat by prefix etc.

Reply:
Since this is just using the KafkaAdmin API, I'd prefer to call it that, but it is easy to change in the future as this would not be used externally.